httpgrpc/httpgrpc.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package httpgrpc;
```

Comment:
Did you make this up, or is this copied from somewhere? I feel like someone ought to have done this already. No matter, I guess.